Packs/MicrosoftECM/Integrations/MicrosoftECM/MicrosoftECM.ps1

```diff
@@ -1027,12 +990,15 @@ Function RestartService()
 		[Parameter()] [string]$serviceName,
 		[Parameter()] [string]$CollectionID,
 		[Parameter()] [string]$CollectionName,
-		[Parameter()] [string]$DeviceName
+		[Parameter()] [string]$DeviceName,
+		[Parameter()] [bool]$ShouldPollResults,
+		[Parameter()] [Int32]$timeoutSeconds
 	)
-	$scriptText = "Get-Service $serviceName | Restart-Service -PassThru -ErrorAction Stop"
+	$escapedServiceName = $serviceName.Replace("`'", "")
```

Comment:
I think what we want to do is escape the `'` and not delete it. Something like:
```suggestion
	$escapedServiceName = $serviceName.Replace("'", "``'")
```

Reply:
Why not - There is no service name with `'` in it and I'm not sure escaping it will work.

Reply:
@hod-alpert if we know that there can't be a service name with `'` in it according to windows naming rules (which I am not 100% sure is the case) then let's fail early instead of changing the user's input.

Reply:
Works for me to create such as service
![image](https://user-images.githubusercontent.com/51712181/100225697-4acb4e80-2f27-11eb-94c0-daf645361b55.png)

![image](https://user-images.githubusercontent.com/51712181/100225720-5454b680-2f27-11eb-9c2c-51053a875ce0.png)

Reply:
Running `Get-Service 'd''b' -ErrorAction Stop`
Seems to work, merging